ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/Decision/DeclineReason.cshtml.cs

```diff
@@ -0,0 +1,128 @@
+using ApplyToBecome.Data.Models.AdvisoryBoardDecision;
+using ApplyToBecome.Data.Services;
+using ApplyToBecomeInternal.Models;
+using ApplyToBecomeInternal.Services;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Primitives;
+using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace ApplyToBecomeInternal.Pages.TaskList.Decision
+{
+	public class DeclineReasonModel : DecisionBaseModel
+	{
+		private readonly ErrorService _errorService;
+
+		public DeclineReasonModel(ErrorService errorService, IAcademyConversionProjectRepository repository, ISession session) : base(repository, session)
+		{
+			_errorService = errorService;
+
+			DeclineOptions = Enum.GetValues(typeof(AdvisoryBoardDeclinedReasons)).Cast<AdvisoryBoardDeclinedReasons>();
+		}
+
+		[BindProperty, MinLength(length: 1, ErrorMessage = "ERROR MESSAGE TBC")]
+		public IEnumerable<string> DeclinedReasons { get; set; }
+
+		[BindProperty] public string DeclineOtherReason { get; set; }
+		[BindProperty] public string DeclineFinanceReason { get; set; }
+		[BindProperty] public string DeclinePerformanceReason { get; set; }
+		[BindProperty] public string DeclineGovernanceReason { get; set; }
+		[BindProperty] public string DeclineChoiceOfTrustReason { get; set; }
+
+		public IEnumerable<AdvisoryBoardDeclinedReasons> DeclineOptions { get; }
+
+		public UIHelpers UI => new UIHelpers(this);
+
+		public async Task<IActionResult> OnGetAsync(int id)
+		{
+			await SetDefaults(id);
+			PreloadStateFromSession(id);
+			SetBackLinkModel(Links.Decision.WhoDecided, id);
+
+			return Page();
+		}
+
+		private void PreloadStateFromSession(int id)
+		{
+			AdvisoryBoardDecision decision = GetDecisionFromSession(id);
+
+			DeclinedReasons = new StringValues(decision.DeclinedReasons?.Select(x => x.ToString()).ToArray());
```

Comment:
This model needs to change as per my previous comment.

![image](https://user-images.githubusercontent.com/109524605/184145336-e689596c-b356-4450-817b-7c608f5ed452.png)